app/views/group_assignment_invitations/_group_assignment_team.erb

```diff
@@ -0,0 +1,31 @@
+<% group_assignment_repo = repo.decorate %>
+
+<div class="one-half left team">
+  <div class="team-panel">
+    <% group_member_count = group_assignment_repo.group.repo_accesses.count %>
+    <%= form_tag({ controller: :group_assignment_invitations, action: :accept_invitation}, :class => "team-actions-form", method: :patch) do %>
+        <%= hidden_field_tag 'group[id]', group_assignment_repo.id %>
+        <% if group_member_count >= group_assignment.max_members %>
+            <%= submit_tag 'Full', class: 'btn', disabled: true %>
+        <% else %>
+            <%= submit_tag 'Join', class: 'btn' %>
+        <% end %>
+    <% end %>
+
+    <span class="team-name" itemprop="name"><%= group_assignment_repo.team_name %></span>
+    <p class="team-description" itemprop="description"><%=group_member_count%> <% if group_member_count != 1%>members<% else %>member<% end %></p>
```

Comment:
`<% if group_member_count != 1%>members<% else %>member<% end %>`

Check out the [pluralize helper](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize).